zeppelin-server/src/main/java/org/apache/zeppelin/rest/LoginRestApi.java

```diff
@@ -135,6 +177,7 @@ private JsonResponse proceedToLogin(Subject currentUser, AuthenticationToken tok
     try {
       logoutCurrentUser();
       currentUser.getSession(true);
+      LOG.info("Try to login");
```

Comment:
delete this, this is only for debugging I believe.

Reply:
done